.travis.yml

```diff
@@ -33,7 +33,7 @@ cache:
 
 notifications:
```

Comment:
Oh, according to the documentation (https://docs.travis-ci.com/user/notifications/), the default is sending notifications to committers. To disable notification, we should use `false` explicitly.

```
By default, email notifications are sent to the committer and the commit author, if they are members of the repository (that is, they have push or admin permissions for public repositories, or if they have pull, push or admin permissions for private repositories).
```